Update docs + S3 bucket pictures (#32)
OpenBB-finance/openbb-docs: content/pro/openbb-copilot.md

### Using your own files

> OpenBB Copilot can also analyze and answer questions using files you provide, such as TXT, PDF, CSV, and XLSX files. To upload a file, click the blue plus icon in the lower right part of your dashboard and select the "Add data" option.

Reviewer: ```suggestion
OpenBB Copilot can also analyze and answer questions using files you provide, such as PDF, CSV, and XLSX files. To add a file to OpenBB Copilot, first [add it to the currently-active dashboard](<link for adding files to Terminal>).
```

Currently TXT isn't supported by the front-end for widgets.

> Once you've uploaded a file, it will become a widget on your dashboard that you can interact with.

Reviewer: ```suggestion
Once the file widget has been added to your dashboard, you can add it to OpenBB Copilot explicitly by clicking the "add widget to context" button, or you can simply begin querying, and OpenBB Copilot will retrieve the data if it is relevant to your query.
```

> OpenBB Copilot can use the data from these widget files to answer your questions. If it does, it will cite the specific files (for PDFs, the exact page) it referenced.

Reviewer: ```suggestion
If OpenBB Copilot uses your file widget to answer any part of you query, it will show citations after its answer.
```
